test/e2e/config/caren.yaml

```diff
@@ -211,7 +211,7 @@ variables:
   # DOCKER_HUB_PASSWORD: ""
 
 intervals:
-  default/wait-controllers: ["3m", "10s"]
+  default/wait-controllers: ["10m", "10s"]
```

Comment:
the helm-repository containers creates PV to store the charts. The PV creation is slow on the nutanix workload cluster. this creates flakes in tests. Increasing timeout to ensure hem-repository comes up.